app/scripts/controllers/logs.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+/* global moment: false */
+
+angular.module('openhimConsoleApp')
+  .controller('LogsCtrl', function ($scope, $location, Api) {
+    function formatLog(log) {
+      return new moment(log.timestamp).format('YYYY-MM-D HH:mm:ss.SSS') + ' - ' + log.level + ': [' + log.label + '] ' + log.message + '\n';
+    }
+
+    var lastFetch;
+    var locParams = $location.search();
+    $scope.params = angular.equals({}, locParams) ? { level: 'info' } : locParams;
+    $scope.logs = '';
+
+    if ($scope.params.from || $scope.params.until) {
+      $scope.autoupdate = false;
+      $scope.tailLogs = false;
+    } else {
+      $scope.autoupdate = true;
```

Comment:
Can you add a control to toggle autoupdate on the interface? It would be useful if you want to study the logs without having it consistently updating. Another idea could be to merge tail and autoupdate? I.e. only autoupdate when tail is true?